From d18c9ac1c7b76a94a9b7ade85f3029c0b27ccb30 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Thu, 17 Sep 2020 23:18:28 -0700
Subject: [PATCH 07/13] QS: Only reduce row count if media player is visible

There's no need to always force qs_less_rows here. The QS layout code
will automatically reduce the amount of rows in order to fit everything
on the screen when the media player is open, so we can still use the
extra space when the media player is not in use.

Change-Id: I99101c6816b402a87d9e429cdcd0be86874bb26b
---
 packages/SystemUI/src/com/android/systemui/qs/TileLayout.java | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/qs/TileLayout.java b/packages/SystemUI/src/com/android/systemui/qs/TileLayout.java
index 694492a3352..0a296bc0188 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/TileLayout.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/TileLayout.java
@@ -48,8 +48,7 @@ public class TileLayout extends ViewGroup implements QSTileLayout {
     public TileLayout(Context context, AttributeSet attrs) {
         super(context, attrs);
         setFocusableInTouchMode(true);
-        mLessRows = (Settings.System.getInt(context.getContentResolver(), "qs_less_rows", 0) != 0)
-                || useQsMediaPlayer(context);
+        mLessRows = (Settings.System.getInt(context.getContentResolver(), "qs_less_rows", 0) != 0);
         updateResources();
 
     }
-- 
2.28.0

